pkg/env/env.go

```diff
@@ -257,7 +258,7 @@ func (e *testEnv) Run(m *testing.M) int {
 	for _, fin := range finishes {
 		// context passed down to each finish step
 		if e.ctx, err = fin.run(e.ctx, e.cfg); err != nil {
-			log.Println(err)
+			log.V(2).ErrorS(err, "ran into an error trying to run the finish action handler")
```

Comment:
Nit: the error prefix string can be more succinct since the log function is already reporting an error, no need to restate it. The prefix could just be the section where the error happened:

```go
log.V(2).ErrorS(err, "Finish action handlers")
```